packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -101,6 +102,7 @@ const initialState: SuiteState = {
         dashboardGraphHidden: false,
         dashboardAssetsGridMode:
             getWindowWidth() < getNumberFromPixelString(variables.SCREEN_SIZE.SM),
+        showDashboardT2B1PromoBanner: true,
```

Comment:
The desktop promo banner is located in a different place currently :( Could you move it to flags? Or better yet, create a separate place for all the promo banners?

Reply:
18027dc62f3da7f2f19331db8afd5b3db821d670
a9a8bb16e5c4a670f9afa5b8fa057d028d233507

Reply:
thanks!!